test/e2e/fixture-builder.js

```diff
@@ -1483,7 +1483,7 @@ class FixtureBuilder {
 
   build() {
     this.fixture.meta = {
-      version: 74,
+      version: 91,
```

Comment:
is this okay for us to bump? It's not clear to me what version this fixture has been left at. It's definitely not 74